recipes-tag/intake-bluesky/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.1.0a8" %}
+{% set setup_py = load_setup_py_data() %}
+
+package:
+    name: 'intake-bluesky'
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/bluesky/intake-bluesky
+    git_rev: v{{ version }}
+
+build:
+    number: 0
+    script: pip install intake-bluesky
+    skip: True  # [py2k]
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - versioneer
```

Comment:
Do we actually need it? All relevant `versioneer` files are incorporated into the repo already.

Reply:
I will remove it